src/universal/modules/teamDashboard/components/ProjectColumn/ProjectColumn.js

```diff
@@ -140,10 +140,6 @@ class ProjectColumn extends Component {
     // the front or back of the list, this will be `undefined`.
     const boundingProject = projects[targetIndex + (before ? -1 : 1)];
     const sortOrder = sortOrderBetween(targetProject, boundingProject, draggedProject, before);
-    const noActionNeeded = sortOrder === draggedProject.sortOrder && draggedProject.status === status;
-    if (noActionNeeded) {
-      return;
-    }
     const updatedProject = {id: draggedProject.id, sortOrder, status};
```

Comment:
As `updateProject` is currently written, the `updatedAt` field isn't changed when just the sortOrder changes. This was from user feedback that dragging things around shouldn't count as an "update". However, if you include `status`, it _will_ count as an update.